app/lib/presenters/complaint.rb

```diff
@@ -0,0 +1,39 @@
+module Presenters
+  class Complaint
+    def initialize(form_builder_payload:)
+      @data = form_builder_payload[:submissionAnswers]
+    end
+
+    def optics_payload
+      {
+        db: 'hmcts',
+        Type: 'Complaint',
+        Format: 'json',
+        RequestMethod: 'Form',
+        RequestDate: @data.fetch(:submissionDate, Date.today),
+        Team: 'INBOX',
```

Comment:
need to set `"Case.ContactMethod": 'Online - gov.uk'`

Reply:
Done